### PR TITLE
test: mender-connect tests: increase timeout.

### PR DESCRIPTION
**Pull request by merlin-northern**
Ticket: None
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar

**Comment by merlin-northern** [comment]
@alfrunes this test also requires that the proxy is connected. (vide your change to keep alive test, and believe or not, but I wanted to get this in here months ago)

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with cherry on top

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with sweet cherry on top

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with sweet cherry on top

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with sweet cherry on top with no stem

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with sweet cherry on top with no stem with frosting

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with sweet cherry on top with no stem with frosting

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with sweet cherry on top with no stem with frosting

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline pretty please please sugar pretty please with sweet cherry on top with no stem with lots of frosting

**Comment by merlin-northern** [comment]
@alfrunes this is not rebased; the error `subprocess.CalledProcessError: Command 'docker network list -q -f name=mender4455758 | xargs -r docker network rm' returned non-zero exit status 123.` was fixed here: https://github.com/mendersoftware/integration/pull/2190

File: tests/tests/test_mender_connect.py [hunk 1]
```diff
@@ -268,6 +270,18 @@ def assert_env(self, docker_env):
...
+        docker_env.device.run(
+            """dbus-send --print-reply --system \\
+              --dest=io.mender.AuthenticationManager \\
+              /io/mender/AuthenticationManager \\
+              io.mender.Authentication1.FetchJwtToken""",
+            wait=proxy_connected_timeout_s,
+        )
+        output = docker_env.device.run(
+            "dbus-send --system --dest=io.mender.AuthenticationManager --print-reply /io/mender/AuthenticationManager io.mender.Authentication1.GetJwtToken"
+        )
```
Comment (alfrunes) [inline tests/tests/test_mender_connect.py:283]: Why do you need the JWT?

**Review by alfrunes — APPROVED** [review]

File: tests/tests/test_mender_connect.py (hunk 1 above)
Comment (merlin-northern) [reply tests/tests/test_mender_connect.py:283]: this is something that (according to my tests) helps in stabilizing the test. I do not need it, but we have to be sure that it is all connected and ready to go before we start this test.

Comment (merlin-northern) [reply tests/tests/test_mender_connect.py:283]: and if you ask about printing it: I want to see for the logging purposes what DBus returned

**Pull request merged**